modules/structs/settings.go

```diff
@@ -14,3 +14,11 @@ type GeneralRepoSettings struct {
 type GeneralUISettings struct {
 	AllowedReactions []string `json:"allowed_reactions"`
 }
+
+// GlobalSettings contains global settings witch dont have its own category exposed by API
+type GlobalSettings struct {
```

Comment:
Why GlobalSettings and not "Attachment" ? I think it should follow the config of /settings/category just like ui, repo

 https://docs.gitea.io/en-us/config-cheat-sheet/#attachment-attachment